plugins/catalog-backend/src/schema/openapi.generated.ts

```diff
@@ -403,7 +400,6 @@ export const spec = {
         required: ['metadata', 'kind', 'apiVersion'],
         description:
           "The parts of the format that's common to all versions/kinds of entity.",
-        additionalProperties: true,
```

Comment:
@eide Thoughts on the above? https://github.com/backstage/backstage/blob/b7540d3fd702a653eaf43d858b16d39f47ff348b/packages/catalog-model/src/schema/Entity.schema.json#L28C24-L28C24